packages/tenants/src/services/__tests__/TenantService.test.ts

```diff
@@ -28,18 +28,20 @@ describe('TenantRecordService', () => {
     jest.clearAllMocks()
   })
 
+  // FIXME: connectionImageUrl is now part of DIDComm module. Tenants records do not currently
```

Comment:
For the new crypto api i've also been looking to solve this problem, and how i tried to solve it is that each module can hook into the the storing process of the tenant config. It's a bit tricky though, as it makes all modules somewhat intertwined with the tenants module ...

Reply:
Well, maybe tenants could become a core feature? :thinking: